lib/builderator/tasks/berkshelf.rb

```diff
@@ -36,11 +36,13 @@ def metadata(cookbook)
       end
 
       desc 'vendor', 'Vendor a cookbook release and its dependencies'
+      option :debug, :type => :boolean, :desc => 'Show debug output'
```

Comment:
Why not make this a `class_option` instead of setting up up individually as a `method_option` everywhere? Removes duplication and means we don't require further changes when we want to use `options[:debug]` somewhere new in the future.